internal/k8s/informers.go

```diff
@@ -48,8 +48,9 @@ func IngressV1Resources() []schema.GroupVersionResource {
 	}
 }
 
-// +kubebuilder:rbac:groups="networking.x-k8s.io",resources=gatewayclasses;gateways;httproutes;backendpolicies;tlsroutes;tcproutes;udproutes,verbs=get;list;watch
-// +kubebuilder:rbac:groups="networking.x-k8s.io",resources=gatewayclasses/status;httproutes/status;backendpolicies/status;tlsroutes/status;tcproutes/status;udproutes/status,verbs=update
+// +kubebuilder:rbac:groups="networking.x-k8s.io",resources=gatewayclasses;gateways,verbs=get;list;watch
+// +kubebuilder:rbac:groups="networking.x-k8s.io",resources=httproutes;backendpolicies;tlsroutes;tcproutes;udproutes,verbs=get;list;watch
```

Comment:
not a big deal but these two lines could be combined since they're the same API group and verbs (would then have to re-gen YAML too)

Reply:
👍